week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -26,7 +26,10 @@ function mood() {
 
 function greaterThan10() {
   const num = 10;
-  const isBigEnough;
+  let isBigEnough = false;
+  if (num >= 10){
+    isBigEnough=true;
+  }
```

Comment:
sorry this point is not very clear, I do not understand what I have to change

Reply:
so variable `isBigEnough` should remain a `const` variable
Remember you write expression to variable assignment, it will be evaluated.
E.g:
```
const colour = "red";
const isBlue = ("blue" === colour");
console.log(isBlue);
```